bpf/enhancedrecording/command.bpf.c

```diff
@@ -40,13 +40,22 @@ struct data_t {
     u64 pid;
     // ppid is the userspace term (i.e task->real_parent->tgid in kernel).
     u64 ppid;
-    char comm[TASK_COMM_LEN];
+    // Command is the executable.
+    char command[TASK_COMM_LEN];
```

Comment:
Can we undo the "cosmetic" changes and move then to a followup PR, perhaps? Or are these actually needed?

Reply:
I looked through the diff again and the most of the purely cosmetic changes are in the bpf C files; I could manually revert the commands and field renames, can't just revert a commit since Jakub originally made a bunch of changes in one big commit. Since the changes in the bpf C files are pretty minimal and easy to reason about IMO I'm leaning towards keeping them as-is.

There are a small handful of spurious style changes I noticed and I think those are mostly due to the fact that I use `gofumpt` to format instead of `go fmt` which is a bit more strict. Again I could undo those too but neither these nor the bpf C changes amount to much of the total diff. What do you think?

Reply:
Can we do the same minor change to `common_data_t` and update the field name from `comm` to `command` then?